tools/state-viewer/src/cli.rs

```diff
@@ -512,3 +532,23 @@ impl ViewTrieCmd {
         view_trie(store, hash, self.shard_id, self.shard_version, self.max_depth).unwrap();
     }
 }
+
+#[derive(Parser)]
+pub struct ViewTrie2Cmd {
+    #[clap(long)]
+    state_root_hash: String,
+    #[clap(long)]
+    shard_id: u32,
+    #[clap(long)]
+    shard_version: u32,
+    #[clap(long)]
+    max_depth: u32,
```

Comment:
Please comment that setting `max_depth` too high isn't recommended for nodes with large state.